test/common_methods_invocations.py

```diff
@@ -830,8 +833,6 @@ def method_tests():
         ('__getitem__', torch.randn(S, S, S), (dont_convert([[0, 2, 3], [1, 3, 3],
                                                              torch.LongTensor([0, 0, 2])]),), 'adv_index_var'),
     ]
-# TODO: clamp with min/max
-
 
```

Comment:
checked the git log and it looks like when this comment was added there was only a single clamp test above.